core/src/Kokkos_View.hpp

```diff
@@ -1749,6 +1749,20 @@ class View : public ViewTraits<DataType, Properties...> {
     m_map   = std::move(rhs.m_map);
     return *this;
   }
+  // -----------------------------------------------------------
+  // public is_assignable
+  // expose underlying view mapping property
+  template <class RT, class... RP>
```

Comment:
I don't see a whole lot of reason to deduce both `RT` and `RP...` here. Just `class... Ts` or something should suffice.